public/search.php

```diff
@@ -99,7 +99,7 @@
                     <div class="input-group-prepend">
                         <span class="input-group-text">NGN</span>
                     </div>
-                    <input type="text" class="form-control investemnt-value-input" placeholder="500,000">
+                    <input type="text" class="form-control investment-value-input" placeholder="500,000" maxlength="10">
```

Comment:
<img width="1248" alt="Screenshot 2020-05-19 at 5 13 32 PM" src="https://user-images.githubusercontent.com/26963369/82351158-203a8f00-99f4-11ea-8a26-1fc6491c6e3a.png">
Great thinking with your fix, Unfortunately, using `maxlength` doesn't solve the maximum value bug as shown in the screenshot above.

Also, the text input still shows NaN when it is empty.

Reply:
I've fixed the `NaN` showing in the input. Still looking for a way around the value tho. `maxlength` is just to make it easier I guess 😅